### PR TITLE
Add support for priority and delay when creating jobs

### PR DESCRIPTION
**Pull request by jansenignacio**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -118,7 +118,16 @@ module.exports = class Beanstalk {
   putJob (data) {
     return new Promise((resolve, reject) => {
       data = JSON.stringify(data)
-      this.client.put(0, 0, data._ttl || (this.defaultTTL * 2), data, function (err, jobid) {
```
Comment (Zmeu213) [inline index.js]: `data` shouldnt contain any "hiden" parameters for `putJob`, it was my mistake to add `_ttl`. Please remove it and add `ttl`, `priority` and `delay` as parameters for `putJob` with default values

**Comment by jansenignacio** [comment]
Updated the signatuure

**Comment by Zmeu213** [comment]
Great, thanks for your collaboration

**Comment by jansenignacio** [comment]
Sure thing! When do you think you'll be able to make a new release to npm?

**Comment by Zmeu213** [comment]
Its already released, you could get it now, check https://www.npmjs.com/package/beanstalk-promises

**Comment by jansenignacio** [comment]
Amazing. Thanks again!

**Pull request merged**